src/provisioner/cluster.py

```diff
@@ -75,6 +77,12 @@ def add_node(self, node: dict):
         self._validate(node)
         self._node_list.append(node)
 
+    @staticmethod
+    def get_machine_id(machine_id: str):
```

Comment:
input name should be node_id to avoid confusion.

Reply:
done